webapp/channels/src/packages/mattermost-redux/src/utils/integration_utils.ts

```diff
@@ -9,7 +9,7 @@ type DialogError = {
     values?: any;
 };
 export function checkDialogElementForError(elem: DialogElement, value: any): DialogError | undefined | null {
-    if (!value && !elem.optional) {
+    if ((!value && value !== 0) && !elem.optional) {
```

Comment:
Maybe what we really want here is `value !== ''`?

Reply:
Since `value` is typed as `any`, it can be `null` or `undefined`, and (I think) those values should be eliminated at this point. So I think it's the best to check all falsy values other than 0 here.
(This condition should return `true` if value is empty, so I think you mean `value === ''`.)

Reply:
You are right I meant `value === ''`. I think this is fine, just one of those cases where I wish we could be more explicit with types. But JS works against us.